src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkTask.java

```diff
@@ -114,7 +114,11 @@ public void put(final Collection<SinkRecord> records) {
 			if (logger.isDebugEnabled()) {
 				logger.debug("Processing record value {} in topic {}", record.value(), record.topic());
 			}
-			writeBatcher.add(sinkRecordConverter.convert(record));
+			if (record.value() != null) {
+				writeBatcher.add(sinkRecordConverter.convert(record));
+			} else {
+				logger.info("Skipping record with null value");
```

Comment:
I am wondering if "warn" is more appropriate here. But I don't know how or why a sink task would receive null, and how unusual that's considered. If it's a common thing that happens a lot and is just natural, then I'd vote for "debug", since then it's just noise to be ignored. But my hunch is that null being received is not a natural event, and thus I lean towards "warn" so it stands out from info logging.